bin/GameIndex.dbf

```diff
@@ -4617,14 +4618,8 @@ Region = NTSC-U
 Serial = SCUS-97150
 Name   = Formula One 2001
 Region = NTSC-U
-Compat = 3
+Compat = 5
 XgKickHack = 1
-[patches = 272F1C14]
-	comment=patches by Plot and Ref Skip videos
-	patch=0,EE,0010aafc,word,00000000
-	patch=0,EE,0010ae2c,word,00000000
-	patch=0,EE,00230010,word,00000000
-[/patches]
```

Comment:
@FlatoutPS2 you said in the forum that it shouldn't be necessary after the most recent commit here (can't embed on mobile)
http://forums.pcsx2.net/Thread-Formula-One-2001-SCES-50004-E--30588?pid=377960&highlight=Formula+One+2001#pid377960